packages/stripe-integration/src/stripe-upe/stripe-ocs-payment-strategy.ts

```diff
@@ -287,21 +274,29 @@ export default class StripeOCSPaymentStrategy implements PaymentStrategy {
         };
     }
 
-    private _getRadioIconSizes(
-        realOuterWidth: string | number = 26,
-        realOuterStrokeWidth: string | number = 1,
-        realInnerWidth: string | number = 17,
-    ) {
+    private _getElementFonts(style?: StripeUPEPaymentInitializeOptions['style']) {
+        if (!Array.isArray(style?.fontsSrc)) {
```

Comment:
It might be easier to understand what is going on in this method if we directly return nothing for all cases:

```suggestion
        if (!style || !style.fontSrc || !Array.isArray(style?.fontsSrc)) {
```

so return might be something like this:
```
return style.fontsSrc.map((cssSrc: string) => ({ cssSrc }));
```